pages/button-group/test.page.tsx

```diff
@@ -3,177 +3,239 @@
 
 import React, { useContext, useState } from 'react';
 
-import Box from '~components/box';
-import Button from '~components/button';
-import ButtonGroup, { ButtonGroupProps } from '~components/button-group';
-import SpaceBetween from '~components/space-between';
-import StatusIndicator from '~components/status-indicator';
+import { Box, Button, ButtonGroup, ButtonGroupProps, Header, SpaceBetween, StatusIndicator } from '~components';
```

Comment:
Aren't we encouraging the direct imports because of the treeshaking? If so, wouldn't it be better to keep it as before since our code base is being used as an example.

Reply:
If we want to enforce this I think we should use an ESLint rule. Rn there are different import styles used across dev pages.